.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] [OPTIONAL] To maintain our git history clean, let's make it a habit of always adding node_modules
      into the `.gitignore` file. What do you think?. 
      The node_modules contain hundreds of megabytes of dependencies. By accidentally 
      committing this file onto your repository can drastically slow down the repository performance.